deno.json

```diff
@@ -0,0 +1,28 @@
+{
+  "tasks": {
+    "dev": "deno test --watch --allow-read"
+  },
+  "license": "MIT",
+  "imports": {
+    "@std/assert": "jsr:@std/assert@1",
+    "@std/expect": "jsr:@std/expect@1",
+    "@std/testing": "jsr:@std/testing@1",
+    "vitest": "./testlib/mod.ts",
```

Comment:
I replaced vite with the Deno testing facilities, mostly to reduce our dependencies.
If vitest does have some nice advantages, we should switch back to it.